Modules/ThirdParty/NIFTI/src/nifti/niftilib/nifti1_io.h

```diff
@@ -109,7 +109,7 @@ typedef enum _analyze75_orient_code {
            nifti1 format definition; it is used to implement one API
            for reading/writing formats in the nifti1 format.
  */
-typedef struct {                /*!< Image storage struct **/
+typedef struct nifti_image_struct {                /*!< Image storage struct **/
```

Comment:
After reading @seanm comment.  I think you could use:

```c++
using nifti_image_struct = struct nifti_image; 
```
to avoid modifying the upstream.

Reply:
Perhaps that does not work with the "foward declaration" ??

Reply:
Thanks for looking into it.